### DISCUSSION TITLE
How to update a file in an archive?

### DISCUSSION THREAD

**Discussion by magicwolf2019** [nmoinvaz/minizip#454] [Q&A] [answered]
Hi!

How to update the contents of an existing file in an archive? Write new data to the end of the file, which is in the previously created archive. Or is it not possible for a archive format zip???

Best regards,
Sergey

**Comment by nmoinvaz** [comment, marked as answer]
Most archive programs will create a new zip on disk and swap old for new. Copies the old zip entries to the new zip and adds the one you are trying to update. If the entry you are updating is at the end of the zip then it might be possible to remove it without creating entirely new zip. but since that is the least likely case nobody does it that way.
